src/components/shared/Grid/SharedGrid.jsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { StyledBox, StyledGrid, StyledItem } from './Grid.styles';
+
+function SharedGrid({
+  items = [],
+  columns = { xs: 4, sm: 8, md: 12 },
+  spacing = { xs: 2, md: 3 },
+  itemSize = { xs: 2, sm: 4, md: 4 },
+  style = {},
```

Comment:
Take your objects, and save them into a styled file that export them with given name